inc/template-tags.php

```diff
@@ -28,245 +37,237 @@ function alcatraz_posted_on() {
 		'<a href="' . esc_url( get_permalink() ) . '" rel="bookmark">' . $time_string . '</a>'
 	);
 
+	$author_id = get_post_field( 'post_author', $post_id );
+
 	$byline = sprintf(
 		esc_html_x( 'by %s', 'post author', 'alcatraz' ),
-		'<span class="author vcard"><a class="url fn n" href="' . esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ) . '">' . esc_html( get_the_author() ) . '</a></span>'
+		'<span class="author vcard"><a class="url fn n" href="' . esc_url( get_author_posts_url( $author_id ) ) . '">' . esc_html( get_the_author_meta( 'nicename', $author_id ) ) . '</a></span>'
```

Comment:
Right now, this is showing the username, not the user's full name, which in most cases, I think would be what we'd want.